### ISSUE TITLE
Allow parameters to be changed on initialization

### ISSUE THREAD

**Issue by ahankinson** [DDMAL/tripoli#19]
The parameters for an `IIIFValidator` instance should be passed in through the `__init__` method.

https://github.com/DDMAL/tripoli/blob/master/tripoli/tripoli.py#L35

``` python
def __init__(self, collect_errors=True, collect_warnings=True, fail_fast=False, unique_logging=True, verbose=False, debug=False)
```

Then:

```
self.collect_errors = collect_errors
```

This should also fix the documentation where it gives two defaults ("None" and whatever is the actual default), e.g., 

> **collect_errors = None**
> Sets whether or not errors are logged. Default True.

**Comment by lexpar** [comment]
The kwargs in init makes sense.

I don't know how to remove the None value in the sphinx autodocs. It looks like it reflects on the class without actually instantiating one (using `getattr`) to get the values of properties. So, I guess I could make them static variables.

**Comment by lexpar** [comment]
Is assigning a default value a valid use of a class attribute in python? I think I remember in Java the `static` keyword was typically used for creating globals or constants within the class. 

I guess as long as the class attributes are all immutable types, it shouldn't cause any weird side effects, which means default boolean options would be ok.

**Comment by ahankinson** [comment]
Yeah, I think it's OK

**Comment by lexpar** [comment]
Fixed in #a1bb923bac10e55c7b50dbd88a533968d6a1c7a6
